### PR TITLE
Add homework3

### PR DESCRIPTION
**Pull request by HuMMaI**
Added third sql homework

### CODE REVIEW

File: Homework3/new_book_catalog.sql [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+SELECT * FROM book_catalog WHERE number_of_pages LIKE '2__';
+
+#Books where price is in interval [20, 40]
+SELECT * FROM book_catalog WHERE price BETWEEN 20 AND 40;
+#Books where price isn`t in interval [10, 30]
+SELECT * FROM book_catalog WHERE price NOT BETWEEN 10 AND 30;
+
+#Deleting books by some criteria
+DELETE FROM book_catalog WHERE id = 5;
+DELETE FROM book_catalog WHERE name = 'Little Boy Lost';
```
Comment (VolodymyrTymtsiv) [inline Homework3/new_book_catalog.sql]: You don't have `name` column in the table.

File: Homework3/new_book_catalog.sql [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+DELETE FROM book_catalog WHERE isbn = 9781967619543;
```
Comment (VolodymyrTymtsiv) [inline Homework3/new_book_catalog.sql]: You need to `set sql_safe_updates = 0;` to execute last 2 statements.

File: Homework3/new_book_catalog.sql (hunk 1 above)
Comment (HuMMaI) [reply Homework3/new_book_catalog.sql]: Done

File: Homework3/new_book_catalog.sql (hunk 2 above)
Comment (HuMMaI) [reply Homework3/new_book_catalog.sql]: Done

**Pull request merged**